setup.py

```diff
@@ -1,24 +1,15 @@
-###############################################################################
-# SKA South Africa (http://ska.ac.za/)                                        #
-# Author: cam@ska.ac.za                                                       #
-# Copyright @ 2013 SKA SA. All rights reserved.                               #
-#                                                                             #
-# THIS SOFTWARE MAY NOT BE COPIED OR DISTRIBUTED IN ANY FORM WITHOUT THE      #
-# WRITTEN PERMISSION OF SKA SA.                                               #
-###############################################################################
-import sys
-
+# Copyright 2013 SKA South Africa (http://ska.ac.za/)
```

Comment:
Not entirely sure what the license is  as I am also adding a license in the https://github.com/ska-sa/katportalclient/pull/50 PR.

Reply:
:man_facepalming: talk about copy and paste, at least the `LICENSE` file is pointed out in the comment.

Reply:
Sorry, I need to perfect my written word. 
I am not entirely sure if the line 
`Copyright 2013 SKA South Africa (http://ska.ac.za/)` is correct. 
As we are no longer `SKA South Africa`  but @ajoubertza answered it over here. 
https://github.com/ska-sa/katportalclient/pull/50/files#r319179863